apps/cloudinary2/src/locations/Field/index.tsx

```diff
@@ -18,22 +18,26 @@ const Field = () => {
   const sdk = useSDK<FieldAppSDK<AppInstallationParameters>>();
   useAutoResizer({ absoluteElements: true });
 
-  const [assets = [], setAssets] = useFieldValue<CloudinaryAsset[]>(sdk.field.id, sdk.field.locale)
+  const [assets = [], setAssets] = useFieldValue<CloudinaryAsset[]>(sdk.field.id, sdk.field.locale);
```

Comment:
Hey! since we know that `assets` could be null here maybe just make the type argument reflect that `CloudinaryAsset[] | null`